libafl/src/corpus/testcase.rs

```diff
@@ -67,6 +69,9 @@ where
     disabled: bool,
     /// has found crash (or timeout) or not
     objectives_found: usize,
+    /// Vector of `Feedback` that deemed this `Testcase` interesting
+    #[cfg(feature = "track_hit_feedbacks")]
+    hit_feedbacks: Vec<Cow<'static, str>>,
```

Comment:
Is it really static?

Reply:
All names return a `Cow<'static, str>` right?
https://github.com/AFLplusplus/LibAFL/blob/7c9bd1157cc49546fbbc05d20ee98512dc9146ab/libafl_bolts/src/lib.rs#L235-L240

Reply:
Ah checks out